api/validator.proto

```diff
@@ -67,11 +74,22 @@ message AuditResponse {
 message ResetRequest {}
 message ResetResponse {}
 
+message GetConstraintRequest {
+  // The name of the constraint to fetch
```

Comment:
what about group/version/kind? We can probably assume the group and version, but kind needs to be part of the request, right?

Reply:
We don't provide the constraint kind as part of the violation data, so we'd need to add that.